components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm

```diff
@@ -104,7 +104,8 @@ $doc.display($prop.getName(), 'view', $fakeObj )##
 #end
 |(% class="pseudoindent" %) |(% class="variant-row-count variant-${variantIndex} variant" %)$count##
 #foreach($propName in $options.defaultProperties)
-|(% class="variant variant-default-input $propName #if ($propName == 'cdna')mandatory#end variant-$variantIndex" %)#if ($xcontext.action == 'edit')$doc.display($propName, 'edit', $variantObj)#else$!{services.rendering.escape($variantObj.getValue($propName), $doc.syntax)}#end##
+|(% class="variant variant-default-input $propName #if ($propName == 'cdna')mandatory#end variant-$variantIndex" %)#if ($xcontext.action == 'edit')$doc.display($propName, 'edit', $variantObj)
+#else#if($propName != 'cdna')$doc.display($propName, 'view', $variantObj)#else$!{services.rendering.escape($variantObj.getValue($propName), $doc.syntax)} #end#end##
```

Comment:
I think we can use the `$doc.display()` in both branches of the `if($propName != cdna)`, but need to test (just to simplify the code)